### ISSUE TITLE
Fix docker-compose error

### ISSUE THREAD

**Issue by noisecapella** [mitodl/cookiecutter-djangoapp#4]
This seems to be introduced by an upgrade to docker-compose where it's less tolerant of the shell injection we do to get the HOST_IP variable.

**Comment by jamiefolsom** [comment]
Yes, I had to remove this line:

```
    export HOST_IP=$(netstat -nr | grep ^0\.0\.0\.0 | awk "{print \$2}") &&
```

Quote all booleans, and hard code the port for the django app:

```
    python manage.py runserver 0.0.0.0:$PORT'
```
becomes:

```    
    python manage.py runserver 0.0.0.0:8070'
```

Then it works with the new docker-machine.

**Comment by noisecapella** [comment]
Fixed by #17
